### PR TITLE
Frontend V1.9.4 (Authentication)

### PR DESCRIPTION
**Pull request by Jgerbrandt**
- Adding auth
- Don't send heartbeat if not authed
- Only admins can write, update, and delete

### CODE REVIEW

File: database/pb_migrations/1717270216_updated_Baro.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}, (db) => {
+  const dao = new Dao(db)
+  const collection = dao.findCollectionByNameOrId("2j2bfgv8i8le9vo")
+
+  collection.createRule = ""
+  collection.updateRule = ""
+  collection.deleteRule = ""
+
+  return dao.saveCollection(collection)
+})
```
Comment (cjchanx) [inline database/pb_migrations/1717270216_updated_Baro.js:20]: What's updated in the telemetry tables? Baro etc.

**Review by cjchanx — APPROVED** [review]

File: RocketControlUnitGUI/src/routes/+page.svelte [hunk 2]
```diff
@@ -12,6 +12,18 @@
 	const modalStore = getModalStore();
 
 	const PB = new PocketBase('http://192.168.0.69:8090');
+	
+	PB.authStore.clear();
+	let auth = false;
+
+	onMount(async () => {
+		const email = import.meta.env.VITE_EMAIL;
```
Comment (cjchanx) [inline RocketControlUnitGUI/src/routes/+page.svelte:20]: do we need to add a blank `.env` into the repo?

File: RocketControlUnitGUI/src/routes/+page.svelte [hunk 3]
```diff
@@ -114,11 +126,13 @@
...
-        intervalId = setInterval(async () => {
-            await PB.collection('Heartbeat').create({
-                message: 'heartbeat'
-            });
-        }, 5000); // 5000 milliseconds = 5 seconds
+		if (auth === true) {
+			intervalId = setInterval(async () => {
+				await PB.collection('Heartbeat').create({
+					message: 'heartbeat'
+				});
```
Comment (cjchanx) [inline RocketControlUnitGUI/src/routes/+page.svelte:133]: Maybe show a quick skeleton alert or something (or make the state transition buttons not show up or something like that) https://www.skeleton.dev/elements/alerts so we know we're not authed in some way

File: RocketControlUnitGUI/src/routes/+page.svelte (hunk 2 above)
Comment (Jgerbrandt) [reply RocketControlUnitGUI/src/routes/+page.svelte:20]: Nah, if you dont have a .env you just wont be authed

File: RocketControlUnitGUI/src/routes/+page.svelte (hunk 3 above)
Comment (Jgerbrandt) [reply RocketControlUnitGUI/src/routes/+page.svelte:133]: Yeah sure

**Pull request merged**